src/scicookie/{{cookiecutter.project_slug}}/build-system/setup.py

```diff
@@ -0,0 +1,17 @@
+# Available at setup time due to pyproject.toml
+from pybind11.setup_helpers import Pybind11Extension
+from setuptools import setup
+
+
+setup(
+    name="OSL Python package",
+    author="{{cookiecutter.author_full_name}}",
+    author_email="{{cookiecutter.author_email}}",
+    url="https://{{ cookiecutter.project_slug }}.com",
+    description="A test project using pybind11",
+    long_description="",
+    #extras_require={"test": "pytest"},
+    # Currently, build_ext only provides an optional "highest supported C++
+    # level" feature, but in the future it may provide more features.
+    python_requires=">=3.7",
```

Comment:
All of this is already specified in `pyproject.toml`. We would need something like this - https://github.com/scientific-python/cookie/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/%7B%25%20if%20cookiecutter.backend%3D%3D'pybind11'%20%25%7Dsetup.py%7B%25%20endif%20%25%7D